check_manifest.py

```diff
@@ -987,6 +992,10 @@ def main():
         global VERBOSE
         VERBOSE = True
 
+    if args.quiet:
+        global QUIET
+        QUIET = True
```

Comment:
One minor point: currently it's possible to enable both --quiet and --verbose at the same time, which will suppress some messages but let other ones through.

I think it would make more sense to allow three modes: quiet, standard, and verbose.  It should be doable with some argparse trickery.  I could do it post merge.

Reply:
Could do it as a mutually_exclusive_group, though I think using both at the same time would be fairly obvious (and intentional)

Reply:
mutually_exclusive_group is probably overkill.  I was thinking something like

```python
    parser.add_argument('--quiet', action='store_const', dest='verbosity', const=0, default=1)
    parser.add_argument('--verbose', action='store_const', dest='verbosity', const=2)
```

(I wish argparse had the opposite of action='count', so each --quiet would cancel exactly one --verbose and vice versa.)

As for "fairly obvious": I've no idea what asking for both verbose and quiet at the same time, intentionally, would mean?  The only situation I can imagine is when one of those was added by some wrapper script, and the other one was passed by the user to countermand it.